README.md

```diff
@@ -55,9 +57,17 @@ docker-compose down -v
 If Optimize, Identity, and Keycloak are not needed you can use the [docker-compose-core.yaml](docker-compose-core.yaml) instead which does not include these components:
 
 ```
-docker-compose -f docker-compose-core.yaml up -d
+docker-compose -f docker-compose-core.yaml up -d -v 
```

Comment:
I guess we had a misunderstanding, the `-v` is a flag which can be added to the docker compose [down](https://docs.docker.com/engine/reference/commandline/compose_down/) command.